scaffolding/lib/utils.js

```diff
@@ -0,0 +1,240 @@
+const
+  fs = require('fs'),
+  ejs = require('ejs'),
+  _ = require('lodash'),
+  { exec } = require('child_process'),
+  path = require('path');
+
+function mkdirp (fullPath) {
+  const parts = fullPath.split(path.sep);
+
+  if (parts.length > 1) {
+    mkdirp(parts.slice(0, parts.length - 1).join('/'));
+  }
+
+  if (! fs.existsSync(fullPath)) {
+    fs.mkdirSync(fullPath);
+  }
+}
+
+function get(store, key) {
```

Comment:
This function doesn't seem to be used. Did I miss something?

Reply:
No, I removed it.